### DISCUSSION TITLE
Module updete

### DISCUSSION THREAD

**Discussion by Mizy78** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#65] [Q&A] [answered]
I cant update modul to 5.0.0. After check still show current version.

**Comment by aMediocreDad** [comment, marked as answer]
Hi, I am assuming you are checking for updates from within Foundry VTT. If you are you might be on a version where updating wouldn't work because of a mistake in the system. 

Try to uninstall the system and reinstall it. It should then update to the latest version.
